app/controllers/idv/enter_password_controller.rb

```diff
@@ -178,13 +178,20 @@ def next_step
       end
     end
 
+    def scrub_message(message)
+      filtered_message = message.dup
+      filtered_message.gsub!(/sponsorID \d+/i, 'sponsorID [FILTERED]')
+      filtered_message
+    end
+
+
     def handle_request_enroll_exception(err)
```

Comment:
Nitpick: Remove extra whitespace here

```suggestion
    def scrub_message(message)
      filtered_message = message.dup
      filtered_message.gsub!(/sponsorID \d+/i, 'sponsorID [FILTERED]')
      filtered_message
    end

    def handle_request_enroll_exception(err)
```

Reply:
Thanks - I believe this was before I ran the linter.